mobsf/DynamicAnalyzer/views/android/dynamic_analyzer.py

```diff
@@ -46,14 +48,17 @@ def dynamic_analysis(request, api=False):
         and_sdk = None
         apks = StaticAnalyzerAndroid.objects.filter(
             APP_TYPE='apk')
+
         for apk in reversed(apks):
+            app_dir = os.path.join(settings.UPLD_DIR, apk.MD5 + '/')
             temp_dict = {
                 'ICON_FOUND': apk.ICON_FOUND,
                 'MD5': apk.MD5,
                 'APP_NAME': apk.APP_NAME,
                 'VERSION_NAME': apk.VERSION_NAME,
                 'FILE_NAME': apk.FILE_NAME,
                 'PACKAGE_NAME': apk.PACKAGE_NAME,
+                'dynamic_report_exists': is_file_exists(os.path.join(app_dir, 'logcat.txt'))
```

Comment:
More nicer if we use pathlib and define logcat path in a variable.

Also please rename `dynamic_report_exists` to something like `DYNAMIC_REPORT_EXISTS` so it matches the case of the rest in the API response.

Reply:
Sure, just from curiosity - 
Why do you want the convention to be capital? 
Capitals are usually for consts ... @ajinabraham

Reply:
Not particularly upper case. It can be either all uppercase or all lowercase. We might already or in the future expose this as a REST api and the response won't look nice with mixed case key names. Just a nit. :)